winui/ComboBox/Highlighting.md

```diff
@@ -0,0 +1,204 @@
+                        DisplayMemberPath="Name" 
+                        IsEditable="true"
+                        IsFilteringEnabled="True"                            
+                        TextHighlightMode="Matched"
+                        TextSearchMode="StartsWith"                           
+                        HighlightedTextForeground="Red"                               
+                        ItemsSource="{Binding Countries}">
+</editors:SfComboBox>
+{% endhighlight %}
+{% highlight C# %}
+
+comboBox.DisplayMemberPath = "Name";
+comboBox.IsEnabled=true;
+comboBox.IsFilteringEnabled = true;
+comboBox.TextHighlightMode = ComboBoxTextHighlightMode.Matched;
+comboBox.TextSearchMode = ComboBoxTextSearchMode.StartsWith;
+comboBox.HighlightedTextForeground = Color.Red;
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+
+![WinUI ComboBox text highlighting based on begining text one character](Highlighting_images/winui-combobox-textsearchmode-startswith-onecharacter.png)
+
+### Searching with more than one character
+
+![WinUI ComboBox text highlighting based on begining text more than one character](Highlighting_images/winui-combobox-textsearchmode-startswith-morethanonecharacter.png)
+
+
+
+
+### **Highlight all occurance of search text**
+It highlights all the matches that contains the typed characters in suggestion list. 
+
+
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfComboBox  x:Name="comboBox"                         
+                DisplayMemberPath="Name” 
+                IsEditable="true"
+                IsFilteringEnabled="True"                             
+                TextHighlightMode="Matched"
+                TextSearchMode="Contains"                           
+                HighlightedTextForeground="Red"                               
+                ItemsSource="{Binding Wonders}">
+</editors:SfComboBox>
+
+{% endhighlight %}
+{% highlight C# %}
+
+        comboBox.DisplayMemberPath = "Name";
+        comboBox.IsEnabled=true;
+        comboBox.IsFilteringEnabled = true;
+        comboBox.TextHighlightMode = ComboBoxTextHighlightMode.Matched;
+        comboBox.TextSearchMode = ComboBoxTextSearchMode.Contains;
+        comboBox.HighlightedTextForeground = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+![WinUI ComboBox text highlighting based on contains one character](Highlighting_images/winui-combobox-textsearchmode-contains-one.png)
+
+### Searching with more than one character
+
+![WinUI ComboBox text highlighting based on contains more than one character](Highlighting_images/winui-combobox-textsearchmode-contains-morethanone.png)
+
+
+## **Highlighting search text mode** 
+ The SfComboBox provides the following modes.
+* **None** - This mode doesnot highlight any text.
+* **Matched** - This mode highlights the text that matches the user input.
+* **Unmatched** - This mode highlights the text that doesnot matches the user input.
```

Comment:
* **Unmatched** - This mode highlights the text that does not match the user input.

Reply:
Addressed all the review changes.